cli/command_policy_show.go

```diff
@@ -160,14 +160,24 @@ func printFilesPolicy(p *policy.Policy, parents []*policy.Policy) {
 func printErrorHandlingPolicy(p *policy.Policy, parents []*policy.Policy) {
 	printStdout("Error handling policy:\n")
 
+	ignoreFile := false
+	if p.ErrorHandlingPolicy.IgnoreFileErrors != nil {
+		ignoreFile = *p.ErrorHandlingPolicy.IgnoreFileErrors
+	}
+
 	printStdout("  Ignore file read errors:       %5v       %v\n",
-		p.ErrorHandlingPolicy.IgnoreFileErrors,
+		ignoreFile,
 		getDefinitionPoint(parents, func(pol *policy.Policy) bool {
 			return pol.ErrorHandlingPolicy.IgnoreFileErrors != nil
 		}))
 
+	ignoreDir := false
+	if p.ErrorHandlingPolicy.IgnoreDirectoryErrors != nil {
+		ignoreFile = *p.ErrorHandlingPolicy.IgnoreDirectoryErrors
```

Comment:
btw, perhaps it's worth exposing helpers on ErrorHandlingPolicy itself, since this code is repeated already?

p.ErrorHandlingPolicy.IgnoreDirectoryErrorsOrDefault(defaultValue)
p.ErrorHandlingPolicy.IgnoreFileErrorsOrDefault(defaultValue)

WDYT ?

Reply:
yeah good idea, I'll give it a shot

Reply:
✔️